core_lib_generator/file_generators/core_lib_class_generator.py

```diff
@@ -112,3 +114,32 @@ def _add_mongo(template_content: str, yaml_data: dict, core_lib_name: str) -> st
             '\n'.join(mongo_conn),
         )
     return updated_file
+
+
+def _add_alembic_funcs(template_content: str, yaml_data: dict, core_lib_name: str) -> str:
+    updated_file = template_content
+    add_func = False
+    for entity in yaml_data:
+        if yaml_data[entity]['migrate']:
+            add_func = True
+    if add_func:
+        imports_list = ['import os', 'import inspect', 'from core_lib.alembic.alembic import Alembic']
+        func_list = []
+        camel_core_lib = snake_to_camel(core_lib_name)
+        static_method_str = '@staticmethod'
+        install_func_str = 'def install(cfg: DictConfig):'
+        upgrade_alembic_str = f'Alembic(os.path.dirname(inspect.getfile({camel_core_lib})), cfg).upgrade()'
+        func_list.append(static_method_str.rjust(len(static_method_str) + 4))

```

Comment:
thank you very much, nice solution but it's complex.  try to make this straightforward and simple. 
another comment. 
4 is repeating itself. remember that when a value repeats more than once it's a variable.? 
tab_size = 4.

BUT this solution is complex. 
try this please

use `\t` instead. read about \t

f`'\t{install_func_str}'`